src/core.c/allomorphs.pm6

```diff
@@ -1,6 +1,14 @@
 my class Allomorph is Str {
     multi method Bool(::?CLASS:D:) { self.Numeric.Bool }
 
+    multi method ACCEPTS(Allomorph:D: Any:D \a) is default {
```

Comment:
Why not multis for Numeric and Str?

Reply:
Because then we wind up with an ambiguous dispatch.